packages/preview/kouhu/0.1.1/README.md

```diff
@@ -0,0 +1,49 @@
+# kouhu (口胡)
+
+`kouku` is a Chinese lipsum text generator for [Typst](https://typst.app). It provides a set of built-in text samples containing both Simplified and Traditional Chinese characters. You can choose from generated fake text, classic or modern Chinese literature, or specify your own text.
```

Comment:
This should  probably be `kouhu` and not `kouku`

Reply:
Oops, thanks for pointing out.